keyboards/preonic/keymaps/trigotometry/keymap.c

```diff
@@ -0,0 +1,204 @@
+#include "preonic.h"
+#include "action_layer.h"
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+#endif
+
+// Layer declarations
+enum preonic_layers {
+  _DVORAK = 0,
+  _QWERTY = 1,
+  _GAMING = 2,
+  _LOWER = 3,
+  _RAISE = 4,
+};
+
+enum preonic_keycodes {
+  DVORAK = SAFE_RANGE,
+  QWERTY,
+  GAMING,
+  LOWER,
+  RAISE,
+};
+
+// QMK predefined macros
+#define SFT_ENT   SFT_T(KC_ENT)   // Tap for Enter, hold for Shift
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  /* Dvorak
+ * ,-----------------------------------------------------------------------------------.
+ * | GEsc |  1   |  2   |  3   |  4   |  5   |   6  |   7  |   8  |   9  |   0  | Del  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |  "   |  ,   |  .   |  P   |  Y   |   F  |   G  |   C  |   R  |   L  | Bksp |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |Shift |  A   |  O   |  E   |  U   |  I   |   D  |   H  |   T  |   N  |   S  |  /   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |SFTENT|  ;   |  Q   |  J   |  K   |  X   |   B  |   M  |   W  |   V  |   Z  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Ctrl | Alt  | GUI  |Lower |    Space    | Raise| Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_DVORAK] = {
```

Comment:
Could you update all of the layers to look more like this: 
https://github.com/qmk/qmk_firmware/blob/1572375bfbdb231c3373a6300fbb7e2ca195d9be/keyboards/preonic/keymaps/default/keymap.c#L95-L101